<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableViewForData">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>521</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{
    background-color: rgb(100, 95, 95);
	font: 900 9pt &quot;Segoe UI&quot;;
    alternate-background-color: rgb(100, 95, 95);
	font-size:15px;
	outline: 0;
}

QHeaderView::section {
    color: white;
	font: 900 9pt &quot;Segoe UI&quot;;	
    background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
} 

QTableView::item:selected
{
	color: white;
	background-color: rgb(64, 64, 64);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(64, 64, 64);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toAdd_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>490</y>
        <width>111</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 900 9pt &quot;Segoe UI&quot;;
	border: 2px solid black;
	background-color: rgb(100, 95, 95);
	font-size: 15px;	
	border-radius: 15px;	
	color:black;
}
QPushButton::hover{
	background-color: rgb(107, 107, 107);
}</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QComboBox" name="distanceSort">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>120</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
  color: black;	
  background-color: rgb(100, 95, 95);
  padding: 10px;
  selection-background-color: rgb(100, 95, 95);
  font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;

}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox{
border: 2px solid black;
	background-color: white;
font: 600 12pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
      </property>
      <item>
       <property name="text">
        <string>Empty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Up to one 100 units</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Up to a 1000 units</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Up to a 10000 units</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Very far(&gt;10000 units)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Sort by distance </string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_button">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>490</y>
        <width>111</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 900 9pt &quot;Segoe UI&quot;;
	border: 2px solid black;
	background-color: rgb(100, 95, 95);
	font-size: 15px;	
	border-radius: 15px;	
	color:black;
}
QPushButton::hover{
	background-color: rgb(107, 107, 107);
}</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>90</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>(one of points)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>160</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Sort by date</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dateSort">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
  color: black;	
  background-color: rgb(100, 95, 95);
  padding: 10px;
  selection-background-color: rgb(100, 95, 95);
  font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;

}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox{
border: 2px solid black;
	background-color: white;
font: 600 12pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
      </property>
      <item>
       <property name="text">
        <string>Empty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Today</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yesterday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>This week</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>This month</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>This year</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Late</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="letterSort">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
  color: black;	
  background-color: rgb(100, 95, 95);
  padding: 10px;
  selection-background-color: rgb(100, 95, 95);
  font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;

}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox{
border: 2px solid black;
	background-color: white;
font: 600 12pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
      </property>
      <item>
       <property name="text">
        <string>Empty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string># (non-letter)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Sort by first letter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Sort by type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeSort">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>330</y>
        <width>211</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
  color: black;	
  background-color: rgb(100, 95, 95);
  padding: 10px;
  selection-background-color: rgb(100, 95, 95);
  font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;

}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox{
border: 2px solid black;
	background-color: white;
font: 600 12pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
      </property>
      <item>
       <property name="text">
        <string>Empty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Another (only 1 in list)</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="add_button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>293</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 900 9pt &quot;Segoe UI&quot;;
	border: 2px solid black;
	background-color: rgb(100, 95, 95);
	font-size: 15px;	
	border-radius: 15px;	
	color:black;
}
QPushButton::hover{
	background-color: rgb(107, 107, 107);
}</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toMain_button">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>490</y>
        <width>111</width>
        <height>34</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 900 9pt &quot;Segoe UI&quot;;
	border: 2px solid black;
	background-color: rgb(100, 95, 95);
	font-size: 15px;	
	border-radius: 15px;	
	color:black;
}
QPushButton::hover{
	background-color: rgb(107, 107, 107);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fillName">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectType">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
  color: black;	
  background-color: rgb(100, 95, 95);
  padding: 10px;
  selection-background-color: rgb(100, 95, 95);
  font: 600 10pt &quot;Yu Gothic UI Semibold&quot;;

}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox{
border: 2px solid black;
	background-color: white;
font: 600 12pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
      </property>
      <item>
       <property name="text">
        <string>Human</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Car</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tree</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Building</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="fillX">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>90</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>X Point</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
      <property name="text">
       <string>Y Point</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fillY">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	font: 600 13pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="ifEmpty_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>473</y>
        <width>431</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
